lib/package.js

```diff
@@ -509,6 +509,12 @@
 
   function loadPackageConfigPath(loader, pkgName, pkgConfigPath) {
     var configLoader = loader.pluginLoader || loader;
+    if (configLoader.execute === false) {
+      var config = configLoader.getConfig();
+      configLoader = new configLoader.constructor();
+      configLoader.config(config);
+      configLoader.execute = true;
+    }
```

Comment:
For example, when setting `loader.execute = false` and using a loader plugin, the same issue will apply.

Actually the issue is really from an API perspective that we assume when setting `loader.execute = false`, the plugin loader will also be set.